simulator/controller/volume/persistentvolume/pvcontroller.go

```diff
@@ -0,0 +1,38 @@
+package controller
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/client-go/informers"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/pkg/controller/volume/persistentvolume"
+	"k8s.io/kubernetes/pkg/volume"
+	"k8s.io/kubernetes/pkg/volume/hostpath"
+	"k8s.io/kubernetes/pkg/volume/local"
+)
+
+func StartPersistentVolumeController(client clientset.Interface, stopCh <-chan struct{}) error {
```

Comment:
This `StartPersistentVolumeController` is needed? Seems duplicated with simulator/controller/pvcontroller.go.